pkg/yurthub/kubernetes/meta/restmapper.go

```diff
@@ -135,16 +151,17 @@ func (rm *RESTMapperManager) deleteKind(gvk schema.GroupVersionKind) error {
 
 // Used to update local files saved on disk
 func (rm *RESTMapperManager) updateCachedDynamicRESTMapper() error {
-	if rm.storage == nil {
-		return nil
-	}
 	rm.RLock()
 	d, err := marshalDynamicRESTMapper(rm.dynamicRESTMapper)
 	rm.RUnlock()
 	if err != nil {
 		return err
 	}
-	return rm.storage.Update(CacheDynamicRESTMapperKey, d)
+	err = rm.storage.Write(rm.cachedFilePath, d)
+	if err != nil {
+		return fmt.Errorf("failed to update cached dynamic RESTMapper, %v", err)
+	}
+	return nil
```

Comment:
how about modify Line160~Line164 to `return rm.storage.Write(rm.cachedFilePath, d)`?

Reply:
Here, we may need error info with more details to specify which component failed, otherwise we can only get the error log like "path does not exist", unknown about which path does not exsit which may make us confused.